src/z3c/etestbrowser/README.txt

```diff
@@ -75,10 +75,10 @@ general TestBrowser use:
   ''
   >>> browser.etree
   Traceback (most recent call last):
-  XMLSyntaxError: ...
+  ValueError: ...
   >>> browser.normalized_contents
   Traceback (most recent call last):
-  XMLSyntaxError: ...
+  ValueError: ...
```

Comment:
I wonder if we want to add a test case with non-blank invalid HTML?  (Is there such a thing?  Browsers accept anything nowadays; I'm less certain about lxml.)

Reply:
This is hard to achieve the way the tests are set up. Using invalid HTML ends in
`PTRuntimeError: Compilation failed` even before the test browser gets the broken HTML. 
On the other hand I do not think that this test is that important as it mostly tests the underlying library and not our code.